app/src/gql/apollo.js

```diff
@@ -134,7 +134,7 @@ export const createApolloProvider = async () => {
     defaultOptions: {
       // apollo options applied to all queries in components
       $query: {
-        fetchPolicy: "cache-and-network",
+        fetchPolicy: "cache-first",
```

Comment:
cache-and-network: This fetch policy will have Apollo first trying to read data from your cache. If all the data needed to fulfill your query is in the cache then that data will be returned. However, regardless of whether or not the full data is in your cache this fetchPolicy will always execute query with the network interface unlike cache-first which will only execute your query if the query data is not in your cache. This fetch policy optimizes for users getting a quick response while also trying to keep cached data consistent with your server data at the cost of extra network requests.

Reply:
This is the theory, but now try it

Reply:
Problem with cache first is, that it will not update. So why does cache-and-network not use existing queries?

Reply:
Ok, will investigate

Reply:
The problem seems to be the `loading` prop. We should remove it if we want the results to be coming from cache: https://github.com/apollographql/react-apollo/issues/2739

Reply:
We can stop relying on loading and only look at `this.balances.length` and so on... I am going to give it a try

Reply:
Nothing... 😞 I was also reading some SmartQueries concrete threads but cannot make it work (get results from cache) other than applying `cache-first`.

Look at this article: https://medium.com/@galen.corey/understanding-apollo-fetch-policies-705b5ad71980

I think if we explicitly have the `updates` (like we in fact do) in our SmartQueries we should be fine actually

Reply:
I can open a Draft PR with the "no-apollo-loading" rewrite but I tested it and only `cache-fist` seems to be really working

Reply:
OK, this is not a good solution. I tried undelegating and had to do a hard refresh in order to be able to see the undelegation in PagePortfolio... So not ideal